### PR TITLE
Put genetic track at top of oncoprint legend

### PR DESCRIPTION
**Pull request by adamabeshouse**
TODO: E2E testing

@inodb @alisman I'm changing package.json, do I need to do anything else with package-lock?

### CODE REVIEW

**Comment by alisman** [comment]
By running npm install, a new package.json should be generated automatically.  You can then commit this package.json.    THere was an issue where on mac, a package called fs would be added, which would then break on linux.  I think @inodb fixed that.

File: src/shared/components/query/QueryStore.ts [hunk 1]
```diff
@@ -1467,7 +1467,7 @@ export class QueryStore
 		if (historyUrl != newUrl)
 			window.history.pushState(null, window.document.title, historyUrl);*/
 
-		formSubmit(urlParams.pathname, urlParams.query);
+		formSubmit(urlParams.pathname, urlParams.query, undefined, "smart");
```
Comment (alisman) [inline src/shared/components/query/QueryStore.ts]: seems like these changes are from other pull request, no?

Comment (adamabeshouse) [reply src/shared/components/query/QueryStore.ts]: done

**Comment by adamabeshouse** [comment]
@alisman having a lot of issues with the screenshot tests, but the effort is ongoing, and in a separate PR https://github.com/cBioPortal/cbioportal/pull/3712 - it safe to merge this, we'll eventually get those screenshots in

**Pull request merged**